SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityJobPoller.java

```diff
@@ -69,6 +72,16 @@ public void runActionOnPoll() {
         continue;
       }
 
+      if (start - taskId.getStartedAt() >= configuration.getTaskExecutionTimeout()) {
```

Comment:
`getWarnIfScheduledJobIsRunningForAtLeastMillis` can be different than our executionTimeout, so I would think that both would still apply. Warn if it's over the first one, kill if it's over the executionTimeout